### ISSUE TITLE
AWS Route53 alias record will be imported with TTL

### ISSUE THREAD

**Issue by chroju** [GoogleCloudPlatform/terraformer#65]
AWS Route53 alias record will be imported to `route53_record.tf` like below.

```hcl
resource "aws_route53_record" "AAAAAAAAA_example-net-_A" {
  alias {
    evaluate_target_health = false
    name                   = "example.net"
    zone_id                = "AAAAAAAAA"
  }

  name    = "example.net"
  ttl     = "0"
  type    = "A"
  zone_id = "${aws_route53_zone.example-net.zone_id}"
}
```

However, `ttl` is not required for alias record.
cf: [AWS: aws_route53_record - Terraform by HashiCorp](https://www.terraform.io/docs/providers/aws/r/route53_record.html)

**Comment by sergeylanzman** [comment]
I think you got error about Conflicts with alias
https://github.com/terraform-providers/terraform-provider-aws/blob/e5073b94c8469d18fefc8382e8b0fd20334621b1/aws/resource_aws_route53_record.go#L83

It's know issue(https://github.com/GoogleCloudPlatform/terraformer/issues/25) terraformer can't get today details about each field. 
Best solution today know delete `ttl   = "0"` from tf files and run `terraform plan`.

**Comment by chroju** [comment]
> It's know issue(#25) terraformer can't get today details about each field.

Exactly. I understand that this issue is to be solved in #25 . Thanks.
